support-frontend/assets/helpers/urls/url.ts

```diff
@@ -75,10 +75,6 @@ function getBaseDomain(): Domain {
 	return DOMAINS.PROD;
 }
 
-function getAbsoluteURL(path = ''): string {
```

Comment:
This was used in one place, when passing the Referrer URL to the `pageView` function, I noted the URLs it returned were invalid in the Ophan events (see below). I've therefore switched to using `document.location.href` as this does the job for collecting the Referrer URL. As there are no remaining cases for `getAbsoluteURL` so I'm removing it.

<img width="647" alt="Screenshot 2024-06-14 at 11 02 49" src="https://github.com/guardian/support-frontend/assets/1590704/08450867-4cb7-42b0-a029-db416c028283">